src/styles/utilities/spacing.js

```diff
@@ -171,6 +171,8 @@ export default {
 
     p0: {
         padding: 0,
+        paddingHorizontal: 0,
+        paddingVertical: 0,
```

Comment:
Just `padding: 0` is sufficient right?

Reply:
RN is a mess. If you use `paddingHorizontal` somewhere, you can't override it with `padding`

Reply:
wow, didn't know that